Cargo.toml

```diff
@@ -35,6 +35,7 @@ gdal = { version = "0.16", default-features = false }
 gdal-sys = "0.9"
 geo = "0.26.0"
 geo-types = { version = "0.7.11", default-features = false }
+geoarrow = { version = "0.3.0" }
```

Comment:
I think we can remove this from the top-level `Cargo.toml` because it's only used by the CLI.